src/commands/activity.js

```diff
@@ -0,0 +1,178 @@
+                },
+                {
+                    "name": "Fishington.io",
+                    "value": "ID: fish",
+                    "inline": true
+                },
+                {
+                    "name": "Chess In The Park",
+                    "value": "ID: chess",
+                    "inline": true
+                },
+                {
+                    "name": "Sketchy Artist",
+                    "value": "ID: artist",
+                    "inline": true
+                },
+                {
+                    "name": "ID: Awkword",
+                    "value": "ID: awkword",
+                    "inline": true
+                },
+                {
+                    "name": "Delete Me Calla",
+                    "value": "ID: delete",
+                    "inline": true
+                },
+                {
+                    "name": "Doodle Crew",
+                    "value": "ID: doodle",
+                    "inline": true
+                },
+                {
+                    "name": "Sketch Heads",
+                    "value": "ID: sketch",
+                    "inline": true
+                },
+                {
+                    "name": "Letter League",
+                    "value": "ID: letter",
+                    "inline": true
+                },
+                {
+                    "name": "Word Snacks",
+                    "value": "ID: word",
+                    "inline": true
+                },
+                {
+                    "name": "SpellCast",
+                    "value": "ID: spell",
+                    "inline": true
+                },
+                {
+                    "name": "Checkers In The Park",
+                    "value": "ID: checkers",
+                    "inline": true
+                },
+                {
+                    "name": "Blazing 8s",
+                    "value": "ID: 8s",
+                    "inline": true
+                },
+                {
+                    "name": "Putt Party",
+                    "value": "ID: putt",
+                    "inline": true
+                },
+                {
+                    "name": "Land-io",
+                    "value": "ID: land",
+                    "inline": true
+                }
+            ],
+            "author": {
+                "name": "Activities",
+                "icon_url": "https://cdn.discordapp.com/emojis/853745991525859338.gif?size=96&quality=lossless"
+            }
+        }
+    ],
+    "attachments": [],
+    "ephemeral": true
+};
+
+module.exports = class extends Command {
+    constructor(client) {
+        super(client, {
+            name: "activity",
+            description: "Start an activity in your voice channel",
+            options: [
+                {
+                    name: "activity",
+                    description: "The ID of the activity to start",
+                    type: "string",
```

Comment:
nein: https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-type

Reply:
Wird gemacht

Reply:
Ist aber nicht die wirkliche "ID" die habe ich manuell zugewisen in activites.json, schauu das help embed an, dann erklärt es sich von selbst